etc/restic/default.env

```diff
@@ -9,7 +9,7 @@
 #  Thus you don't have to provide all the arguments like
 #  $ restic --repo ... --password-file ...
 
-source /etc/restic/_global.env
+source $INSTALL_PREFIX/etc/restic/_global.env
```

Comment:
This is the prime example of why we need to `sed` the files when a user installes with a custom `$PREFIX`. Without this substitution, this file will always look at `/etc/restic/global.env` which is not correct if the user installed them to `/usr/local/etc/restic/_global.env`.

Reply:
Do you think the following could work? I've recently used something like this to do relative sourcing.

```shell
source $(dirname "$0")/_global.env
```

I think it _could work_ as long as the relative path to `_global.env` is specified, in this case, they both sit at the same dir.

In case this could work, it could help to simplify the Makefile, with all the copies and `sed`. But I'm not very sure if could work or cause any problem.

I tried it and if you do:

```console
$ source /path/to/default.env
```
it seems to source `_global.env` correctly.

Reply:
EDIT: Ouch!! I think you can forget the above comment, as I realize that the substitution is in many other places and not only affects how environment files are sourced.

Reply:
In this case, it would have worked just fine!

But the issue is just like you found out that this is a bit more cumbersome to do in places like in the `restic-backup@.service` and so